salt/kubelet/init.sls

```diff
@@ -82,6 +83,7 @@ kubelet:
       - file:   /etc/kubernetes/manifests
       - file:   /etc/kubernetes/kubelet-initial
       - kubelet-config
+      - cmd: unmount-swaps
```

Comment:
Wouldn't it be better to move this requirement here? https://github.com/kubic-project/salt/blob/c3b81a6ab211e84209056a3f687fb818b2aa785e/salt/kubelet/init.sls#L71, on the `service.running` statement?

Reply:
The swap has to be disabled before the service is started, is that the right location to express this requirement?

Reply:
Sorry, this diff view confused me. Everything looks fine for me.